hack/run-integration-tests.sh

```diff
@@ -103,6 +103,7 @@ trap "cleanup" EXIT
 sudo chmod 600 "${KUBECONFIG}"
 
 prepare_namespaces
-helm_install
-
+if [[ "${job_name}" == "target-browser" ]]; then
```

Comment:
add similar condition for the line `trap "cleanup" EXIT` present in this script.. Check if works fine.